reqs/requirements-dev-lite.txt

```diff
@@ -18,7 +18,10 @@ pytest-order>=1.2.1
 pytest-random-order>=1.1.1
 pytest-timeout>=2.3.1
 pytest-xdist>=3.6.1
-requirements-parser>=0.2.0
+requirements-parser>=0.9.0
 responses>=0.23.1 # requests mocking
-snapshottest==0.6.0 # GX Cloud atomic renderer tests
+setuptools>=70.0.0
```

Comment:
`package_resources` is part of setuptools, and not installed by default for Python 3.12.